### ISSUE TITLE
Package summary contains markdown instead of first plain text sentence

### ISSUE THREAD

**Issue by tiguchi** [verdaccio/verdaccio#2748]
**Your Environment**
 * **verdaccio version**: 5.2.2
 * **node version**: v12.22.7
 * **package manager**: npm@6.14.15
 * **os**: Debian 10
 * **platform**: npm

**Describe the bug**

I created a `README.md` in the root directory of an NPM package project that contains a build status badge as Markdown clickable image tag of the following format:

```markdown
# Project Name
[![Build Status](https://some.url/build/status)](https://some.url/path/to/detail/page)

Project summary
```

The readme appears correct on the package's detail page. But the project summary contains only the unfiltered mardkown of the build status badge:

![2021-12-10_13-40](https://user-images.githubusercontent.com/6569525/145644735-ff3d630c-40c6-4e0a-a6ac-2474d70d8163.png)

**Expected behavior**

I expect the actual summary (first plain text sentence of the readme) to appear instead:

![2021-12-10_13-42](https://user-images.githubusercontent.com/6569525/145644982-2ed4b3ee-fe73-44e0-bd40-467f0a6faabb.png)

**Comment by tiguchi** [comment]
I just realized that `package.json` has support for a "description" field, which was missing in my affected project. I added that missing property, deployed a new version of the package to Verdaccio and the correct description text is showing now.

So this is really a minor problem with a good workaround. Feel free to close this ticket

**Comment by juanpicado** [comment]
So, if you don't include the description field on publish the readme became description by default? 🤔

**Comment by tiguchi** [comment]
> So, if you don't include the description field on publish the readme became description by default? 🤔

That's correct, and It seems to be working pretty well, except for that image markup glitch. The fallback even skips the first heading line (`# Title`) and goes straight to the first sentence of the readme

**Comment by mbtools** [comment]
It's a npm feature. Npm is using [`normalize-package-data`](https://www.npmjs.com/package/normalize-package-data) which will fill an empty description from the readme.

There's a related issue about badges at the top of the readme 😀 
https://github.com/npm/normalize-package-data/issues/89

@tiguchi, feel free to create a PR which fixes this problem [here](https://github.com/npm/normalize-package-data/blob/main/lib/extract_description.js).
